doc/user/bgp.rst

```diff
@@ -2827,6 +2827,12 @@ address-family:
    The CLI will disallow attempts to configure incompatible leaking
    modes.
 
+It is possible to not retain VPN prefixes that are not imported by local VRF
```

Comment:
Let's put the command description after the command.  This is how it's normally done.

Reply:
ok. added after the command.